lib/daru.rb

```diff
@@ -72,6 +74,14 @@ def plotting_library= lib
         raise ArgumentError, "Unsupported library #{lib}"
       end
     end
+
+    def error_stream= stream 
+      class_variable_set(:@@error_stream, stream)
+    end
+
+    def error_stream
+      @@error_stream
+    end
```

Comment:
Why did you decided to go with class variable? Instance variable of `Daru` module would be enough -- with those two methods becoming just `attr_accessor :error_stream`